.circleci/config.yml

```diff
@@ -182,6 +187,19 @@ jobs:
     - run:
```

Comment:
After `- *attach_workspace` add:
```
- attach_workspace:
      at: ~/
```
and I think that will restore the workspace you cached!